plugins/org.jkiss.dbeaver.ui.navigator/src/org/jkiss/dbeaver/ui/navigator/database/DatabaseNavigatorLabelProvider.java

```diff
@@ -233,6 +245,13 @@ public String getToolTipText(Object element) {
         return null;
     }
 
+    private void addTipToText(@NotNull StringBuilder info, @NotNull String label, @NotNull String parameter) {
+        info.append(label)
+            .append(": ")
+            .append(parameter)
+            .append("\n");
+    }
```

Comment:
I'd prefer to bind values to labels through the `NLS#bind`. Some languages have really goofy sentence structures.

Reply:
![2022-11-18 13_10_48-DBeaver  C__Users_anvolka_work_DbeaverP_dbeaver-workspace-idea  - C__Users_anvol](https://user-images.githubusercontent.com/45152336/202678381-544796b0-1a89-4297-b40d-7619fe729811.png)
Brr.

![2022-11-18 13_06_16-DBeaver  C__Users_anvolka_work_DbeaverP_dbeaver-workspace-idea  - C__Users_anvol](https://user-images.githubusercontent.com/45152336/202678422-a6343831-293c-4f01-8573-f951f821f910.png)
Also brr.

What for?

Reply:
See my previous comment about newlines